### PR TITLE
Don't use default image for CanvasSaveButton

### PR DESCRIPTION
**Pull request by oterral**
I think it  makes it more easier to setup CSS for button in the application.

### CODE REVIEW

**Comment by edellucien** [comment]
Netlify deploy fails however @oterral

**Comment by oterral** [comment]
you can rereview @edellucien

File: src/components/CanvasSaveButton/CanvasSaveButton.js [hunk 1]
```diff
@@ -131,11 +141,34 @@ class CanvasSaveButton extends PureComponent {
...
+
         if (extent) {
-          const pixelTopLeft = map.getPixelFromCoordinate(getTopLeft(extent));
-          const pixelBottomRight = map.getPixelFromCoordinate(
-            getBottomRight(extent),
-          );
+          firstCoordinate = getTopLeft(extent);
+          oppositeCoordinate = getBottomRight(extent);
+        } else if (coordinates) {
+          // In case of coordinates coming from DragBox interaction:
```
Comment (gevos) [inline src/components/CanvasSaveButton/CanvasSaveButton.js]: This explanation would be nice at the prop description

**Comment by gevos** [comment]
Except for the small docu comment, LGTM

File: src/components/CanvasSaveButton/CanvasSaveButton.js (hunk 1 above)
Comment (oterral) [reply src/components/CanvasSaveButton/CanvasSaveButton.js]: It's comment for developers. A user does not care, he will send an array of coordinates and expects it works.

**Pull request merged**